File: bronze/9063/c++/main.cpp
#include <iostream>

int main()
{
    int num, x, y;

    int minX = 100000, minY = 100000, maxX = -10000, maxY = -10000;
    std::cin >> num;
    for (int i = 0; i < num; i++) {
        std::cin >> x >> y;

        minX = (minX > x) ? x : minX;
        minY = (minY > y) ? y : minY;
        maxX = (maxX < x) ? x : maxX;
        maxY = (maxY < y) ? y : maxY;

    }

    std::cout << ((maxX - minX) * (maxY - minY));
        


    return 0;
}

File: bronze/9063/python/main.py

if (__name__ == "__main__"):
    num = int(input())
    minX, minY = 100000, 100000
    maxX, maxY = -100000, -100000

    for i in range(num):
        x, y = map(int, input().split())
        if (minX > x): minX = x
        if (minY > y): minY = y
        if (maxX < x): maxX = x
        if (maxY < y): maxY = y
        
    print((maxX - minX) * (maxY - minY))